kafka_consumer/check.py

```diff
@@ -246,6 +246,7 @@ def _validate_consumer_groups(self, val):
 """)
 
     def _send_event(self, title, text, tags, type, aggregation_key):
+        """Emit an event to the Datadog Event Stream."""
```

Comment:
I forgot doing something when I implemented this, I think the prototype should be:
```
def _send_event(self, title, text, tags, type, aggregation_key, severity='info')
```

and the event_dict below should look like this:
```
event_dict = {
   ...
   alert_type=severity,
   ...
}
```

And when we call `_send_event()` from the negative `consumer_lag` condition above in line 202, we should set severity='warn' or 'error'.

Could you fix that for me? 😊  Otherwise we probably want to open a PR to do that :) 

By all means, make me do it if you don't have the time.